plugins/trace_api_plugin/configuration_utils.cpp

```diff
@@ -7,15 +7,15 @@
 namespace eosio::trace_api::configuration_utils {
    using namespace eosio;
 
-   chain::abi_def abi_def_from_file(const std::string& file_name, const fc::path& data_dir )
+   chain::abi_def abi_def_from_file(const std::string& file_name, const std::filesystem::path& data_dir )
    {
       fc::variant abi_variant;
-      auto abi_path = fc::path(file_name);
+      auto abi_path = std::filesystem::path(file_name);
       if (abi_path.is_relative()) {
          abi_path = data_dir / abi_path;
       }
 
-      EOS_ASSERT(fc::exists(abi_path) && !fc::is_directory(abi_path), chain::plugin_config_exception, "${path} does not exist or is not a file", ("path", abi_path.generic_string()));
+      EOS_ASSERT(std::filesystem::exists(abi_path) && !std::filesystem::is_directory(abi_path), chain::plugin_config_exception, "${path} does not exist or is not a file", ("path", abi_path.generic_string()));
```

Comment:
Thoughts on adding a `to_variant(fs::path)` so we don't need to put `generic_string()` in all these log/asserts?

Reply:
+1. Although a new logging framework is likely going to make us add `fmt::formatter<>` for all our types.

Reply:
Looks like `to_variant` is already provided.